app/main.py

```diff
@@ -1,13 +1,48 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
+    return {
+        "errors":
+            [
+                {
+                    "line": error_item["line_number"],
+                    "column": error_item["column_number"],
+                    "message": error_item["text"],
+                    "name": error_item["code"],
+                    "source": "flake8"
+                }
```

Comment:
it is the code duplication, you can reuse already implemented function, for example, `format_linter_error`